### DISCUSSION TITLE
Stellarium old version

### DISCUSSION THREAD

**Discussion by Max55max** [Stellarium/stellarium#3603] [General]
Hi,
Where  can I download old Stellarium version? I have mac OS High Sierra(10.13) and it doesn’t support the new version.
Thank you
Best regards
Massimo Calabresi

**Comment by gzotti** [comment]
You find all releases by following the "all releases" link on stellarium.org. (right panel)

**Reply by Max55max** [reply]
Thank you
Massimo
